packages/jest-runner/fuzz.ts

```diff
@@ -63,6 +63,18 @@ export function fuzz(
 		// affecting other tests, e.g. set a test specific timeout.
 		const localConfig = JSON.parse(JSON.stringify(fuzzingConfig));
 
+		if (process.env.JAZZER_LIST_FUZZTEST_NAMES != undefined) {
```

Comment:
`if (process.env.JAZZER_LIST_FUZZTEST_NAMES) {...}` would check all falsy values including `null` and `""`.